### PR TITLE
Update locale for profile page

### PR DESCRIPTION
**Pull request by lpwanw**

### CODE REVIEW

File: app/models/profile.rb [hunk 1]
```diff
@@ -3,10 +3,8 @@
 class Profile < ApplicationRecord
   belongs_to :user
 
-  validates :name, presence: true
   validates :name, length: { maximum: 100 }
```
Comment (nguyenphuhongloan) [inline app/models/profile.rb:6]:
```suggestion
  validates :name, length: { maximum: 50 }
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,31 +1,220 @@
...
+    abbr_day_names:
+      - Sun
+      - Mon
+      - Tue
+      - Wed
+      - Thu
+      - Fri
+      - Sat
+    abbr_month_names:
+      -
```
Comment (nguyenphuhongloan) [inline config/locales/en.yml:20]: remove this line

File: config/locales/en.yml [hunk 3]
```diff
@@ -1,31 +1,220 @@
...
+      - Wednesday
+      - Thursday
+      - Friday
+      - Saturday
+    formats:
+      default: "%Y-%m-%d"
+      long: "%B %d, %Y"
+      short: "%b %d"
+    month_names:
+      -
```
Comment (nguyenphuhongloan) [inline config/locales/en.yml:46]: remove this line

File: config/locales/vi.yml [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+    abbr_day_names:
+      - CN
+      - Thứ 2
+      - Thứ 3
+      - Thứ 4
+      - Thứ 5
+      - Thứ 6
+      - Thứ 7
+    abbr_month_names:
+      -
```
Comment (nguyenphuhongloan) [inline config/locales/vi.yml:20]: remove this line

File: config/locales/vi.yml [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+      - Thứ tư
+      - Thứ năm
+      - Thứ sáu
+      - Thứ bảy
+    formats:
+      default: "%d-%m-%Y"
+      long: "%d %B, %Y"
+      short: "%d %b"
+    month_names:
+      -
```
Comment (nguyenphuhongloan) [inline config/locales/vi.yml:46]: remove this line

File: config/locales/vi.yml [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+      not_a_number: không phải là số
+      not_an_integer: phải là một số nguyên
+      odd: phải là số lẻ
+      other_than: cần phải khác %{count}
+      present: cần phải để trắng
+      taken: đã tồn tại
+      too_long: quá dài (tối đa %{count} ký tự)
+      too_short: quá ngắn (tối thiểu %{count} ký tự)
+      wrong_length: độ dài không đúng (phải là %{count} ký tự)
+      required: "phải có"
```
Comment (nguyenphuhongloan) [inline config/locales/vi.yml:109]: remove ""

File: config/locales/vi.yml (hunk 5 above)
Comment (lpwanw) [reply config/locales/vi.yml:46]: This is for month zero, which does not exist. ✌🏽

File: app/models/profile.rb (hunk 1 above)
Comment (lpwanw) [reply app/models/profile.rb:6]: @nguyenphuhongloan Why is the name limited to only 50 characters?

File: config/locales/vi.yml (hunk 5 above)
Comment (lpwanw) [reply config/locales/vi.yml:46]: The same goes for the lines above.

File: config/locales/vi.yml (hunk 6 above)
Comment (lpwanw) [reply config/locales/vi.yml:109]: Thank you!!

File: app/models/profile.rb (hunk 1 above)
Comment (nguyenphuhongloan) [reply app/models/profile.rb:6]: 50 is enough

Comment (lpwanw) [reply app/models/profile.rb]: @nguyenphuhongloan oke boss

File: config/locales/vi.yml (hunk 5 above)
Comment (nguyenphuhongloan) [reply config/locales/vi.yml:46]: what is month zero?

Comment (lpwanw) [reply config/locales/vi.yml:46]: @nguyenphuhongloan 
It does not exist. This line is for better mapping between a month and its locale, e.g., month 1 for Jan. So we do not have to subtract 1 from the month.

**Pull request merged**